build/VSIX.targets

```diff
@@ -1,13 +1,14 @@
 <Project>
   <PropertyGroup>
-    <PackageDependsOn Condition="'$(OS)'=='Windows_NT'">$(PackageDependsOn);GenerateVSIX</PackageDependsOn>
+    <RestoreDependsOn Condition="'$(OS)'=='Windows_NT'">$(RestoreDependsOn);RestoreVSIX</RestoreDependsOn>
+    <PackageDependsOn Condition="'$(OS)'=='Windows_NT'">$(PackageDependsOn);PackageVSIX</PackageDependsOn>
     <VSIXName>Microsoft.VisualStudio.RazorExtension</VSIXName>
     <VSIXProject>$(RepositoryRoot)tooling\$(VSIXName)\$(VSIXName).csproj</VSIXProject>
   </PropertyGroup>
 
   <Target
     Name="GenerateVSIX"
```

Comment:
Where is this target called now? Is this obsolete?

Reply:
Or is this just left here as a "meta-target"

Reply:
I left it as a 'meta-target', as CLI sugar if you only want to build the vsix project. I can remove it you'd rather clean it out.